packages/sanity/src/core/form/components/FormInput.tsx

```diff
@@ -75,7 +65,7 @@ export const FormInput = memo(function FormInput(
 /**
  * An input that takes input props for object or array and renders an input for a given sub-path
  */
-const FormInputInner = memo(function FormInputInner(
```

Comment:
The memo here increased render work unnecessarily since the parent `FormInput` already has `React.memo`. Which means that whenever any props change in a way that causes `FormInput` to render it'll always lead to `FormInputInner` rerendering. There isn't a case where `FormInput` would render and `FormInputInner` using `memo` would allow it to skip a render, and so the props comparison it ends up doing on each render is useless busywork.